testData/deviceReporter/installedSolutions.json

```diff
@@ -113,6 +113,10 @@
 
     {
         "id": "com.android.settings.system"
+    },
+
+    {
+        "id": "net.gpii.browserExtension"
```

Comment:
this file is being used by default with GPII and to avoid major explosions when someone installs GPII on a computer without having all the supported AT installed, we've decided to only put the built in solutions in this file.. Hence the browserExtension should not be here